csharp/extractor/Semmle.Extraction.CSharp/Entities/UserOperator.cs

```diff
@@ -161,6 +162,9 @@ public static bool OperatorSymbol(string methodName, out string operatorName)
                 case "op_False":
                     operatorName = "false";
                     break;
+                case var @checked when Regex.IsMatch(@checked, "^op_Checked.*"):
+                    operatorName = @checked;
+                    break;
```

Comment:
Why is this change needed? Are there any tests that cover it? `checked +` might be another operator name that we could use. That would match better the unchecked `+`.

Reply:
Indeed; we should use `checked +` instead. Apparently some existing tests exercise this; probably because checked operators exist in library code.